include/seastar/websocket/server.hh

```diff
@@ -60,6 +208,15 @@ class connection : public boost::intrusive::list_base_hook<> {
     std::unique_ptr<reply> _resp;
     queue<std::unique_ptr<reply>> _replies{10};
     bool _done = false;
+
+    websocket_parser _websocket_parser;
+    queue <temporary_buffer<char>> _input_buffer;
```

Comment:
nitpick: in our code style type sticks to template parameters: `queue<temporary_buffer<char>>`. Same for the queue in line 216 below